crates/re_selection_panel/src/visualizer_ui.rs

```diff
@@ -50,8 +50,27 @@ pub fn visualizer_ui(
     .hover_text("Add additional visualizers")
     .disabled_hover_text("No additional visualizers available");
 
+    let markdown = "# Visualizers
```

Comment:
great, love it! we can re-use some of this on the doc page we still haven't written on this

Reply:
I thought so 😅

Reply:
BTW, these tooltip will very much benefit from links to the docs, when those exists.